se2csv.py

```diff
@@ -36,37 +44,68 @@ def closeInput(dataFile):
 def openOutFile(fileName, writeMode="w"):
     if fileName != "":
         return open(fileName, writeMode)
-
-# open the output files
-def openOutFiles(invFileName, optFileName):
-    invFile = openOutFile(invFileName, writeMode)
-    optFile = openOutFile(optFileName, writeMode)
-    return (invFile, optFile)
     
 # close output files        
-def closeOutFiles(invFile, optFile):
-    if invFile:
-        invFile.close()
-    if optFile:
-        optFile.close()
+def closeOutFiles(devsFile):
+    for devFile in devsFile.itervalues():
+        devFile.close()
 
 # write output file headers
 def writeHeaders(outFile, items):
     outFile.write(delim.join(item for item in items)+"\n")
 
 # write data to output files
-def writeData(msgDict, invFile, optFile):
-    global invSeq, optSeq
-    if invFile:
-        if headers and (invSeq == 0) and (msgDict["inverters"] != {}):
-            writeHeaders(invFile, invItems)
-        for seId in msgDict["inverters"].keys():
-            invSeq = writeDevData(invFile, invOutFmt, msgDict["inverters"][seId], invItems, invSeq)
-    if optFile:
-        if headers and (optSeq == 0) and (msgDict["optimizers"] != {}):
-            writeHeaders(optFile, optItems)
-        for seId in msgDict["optimizers"].keys():
-            optSeq = writeDevData(optFile, optOutFmt, msgDict["optimizers"][seId], optItems, optSeq)
+def writeData(msgDict, devsFilePrefix):
+    global devsSeq, devsFile
+    if devsFilePrefix:
+        if msgDict != {}:
```

Comment:
it looks like unwrap_metricsDict() iterates cleanly so this check is unnecessary